src/docfx/serve/lsp/LanguageServerHost.cs

```diff
@@ -39,6 +40,10 @@ internal class LanguageServerHost
                     .AddSingleton<DiagnosticPublisher>()
                     .AddSingleton<LanguageServerCredentialProvider>()
                     .AddSingleton<LanguageServerBuilder>()
+                    .AddSingleton(new ConfigurationItem
```

Comment:
What's this for?

Reply:
Get the extension config like trace level